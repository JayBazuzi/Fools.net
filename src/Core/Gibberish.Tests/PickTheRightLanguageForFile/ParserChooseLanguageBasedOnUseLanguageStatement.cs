using ApprovalTests;
using Gibberish.Tests.ZzTestHelpers;
using NUnit.Framework;

namespace Gibberish.Tests.PickTheRightLanguageForFile
{
	[TestFixture]
	public class ParserChooseLanguageBasedOnUseLanguageStatement
	{
		[Test]
		public void just_a_uselanguage_statement_should_be_empty_parse()
		{
			var input = "use language fasm\r\n";
			var subject = new ParseFasm();
			var result = subject.GetMatch(input, subject.FasmFile);
			Approvals.VerifyJson(result.PrettyPrint());
		}
	}
}
