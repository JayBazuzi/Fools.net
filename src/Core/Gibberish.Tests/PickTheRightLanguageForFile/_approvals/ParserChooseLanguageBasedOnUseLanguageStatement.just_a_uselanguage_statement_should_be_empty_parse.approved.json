{
    "Declarations": [],
    "Statements": [],
    "MaybeName": null,
    "Errors": []
}